Critical: Develop now failing Travis tests on OSX

Well this is embarrassing .... 

Sometime after commits of #378 and then #390, develop started failing Travis build tests for the OSX docker.  Reverting #390 does not fix the issue.  I haven't tried reverting #378, but it is not clear this will help either.  Both passed all CI tests at the time they were merged.

Travis log reports:
```
CMake Error in CMakeLists.txt:
  Imported target "sidre" includes non-existent path
    "/usr/local/Cellar/open-mpi/4.0.0/include"
  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
  * The path was deleted, renamed, or moved to another location.
  * An install or uninstall procedure did not complete successfully.
  * The installation package was faulty and references files it does not
  provide.
```
Which looks like an issue with sidre paths.  I am hesitant to start reverting until we have a clear picture of what is going on, hence this discussion.

--- a/.travis.yml
+++ b/.travis.yml
@@ -74,8 +74,8 @@ script:
       cd ..;
       mkdir thirdPartyLibs;
       cd thirdPartyLibs;
-      wget https://www.dropbox.com/s/dd7hvw2hfmws5ax/tpl-osx-clang10.tar.gz;
-      tar -xf tpl-osx-clang10.tar.gz;
+      wget https://www.dropbox.com/s/erttwgj8yclo9gx/tpl-osx-clang10-20190416.tar.gz;
+      tar -xf tpl-osx-clang10-20190416.tar.gz;
       cd ../GEOSX;
       bash ${TRAVIS_BUILD_DIR}/scripts/travis_build_and_test.sh;
     else

